src/main/resources/example/repo/test/bin.yaml

```diff
@@ -0,0 +1,7 @@
+repo:
+  type: file
+  storage: default
+  permissions:
+    "test":
```

Comment:
@g4s8 is it actually necessary to add quotes around `test`?

Reply:
No, it's not necessary.

Reply:
It's necessary when you type `#:->|$%&{}[]` in the key.